frappe/public/js/frappe/list/list_view.js

```diff
@@ -903,7 +903,7 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 			return this.settings.get_form_link(doc);
 		}
 
-		const docname = doc.name.match(/[%'"#\s]/)
+		const docname = doc.name.match(/[%'"#\\/\s]/)
 			? encodeURIComponent(doc.name)
 			: doc.name;
```

Comment:
```suggestion
		const docname = encodeURIComponent(doc.name)
```

can't we directly do this? will this cause any problems?

`frappe.utils.get_form_link` encodes the name directly as well :P
https://github.com/frappe/frappe/blob/3bf154c7ce473a6b1ad9e536242e3071fa4d9466/frappe/public/js/frappe/utils/utils.js#L843-L845

Reply:
LGTM

Reply:
LGTM